### PR TITLE
Preparing release 3.0.0

### PR DESCRIPTION
**Pull request by Natim**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
 
 This document describes changes between each past release.
 
-2.5.0 (unreleased)
+3.0.0 (2018-10-17)
 ------------------
 
+**Breaking changes**
+
+- Upgrade to kinto-http.py 10.0 means that the batch will fails on
```
Comment (leplatrem) [inline CHANGELOG.rst]: will fail

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1,11 +1,21 @@
...
+  4xx, use the ``--ignore-batch-4xx`` to keep the previous behaviour.
```
Comment (leplatrem) [inline CHANGELOG.rst]: if one of the server responses has a 4XX status

File: CHANGELOG.rst [hunk 3]
```diff
@@ -1,11 +1,21 @@
...
+
+**New features**
+
+- Add an ``--ignore-batch-4xx`` option to explicitly ask for silent
+  4xx errors.
```
Comment (leplatrem) [inline CHANGELOG.rst]: What about datetime?

Comment (Natim) [reply CHANGELOG.rst]: It is in the bugfixes part.

**Pull request merged**